swarm/network/stream/streamer_test.go

```diff
@@ -39,6 +39,20 @@ func TestStreamerSubscribe(t *testing.T) {
 	}
 }
 
+func TestStreamerRequestSubscription(t *testing.T) {
```

Comment:
i see no textExchange tests for request subscription msg exchange.

Reply:
is this not addressed by `TestStreamerRequestSubscriptionQuitMsgExchange` in `swarm-network-rewrite`?